.rubocop.yml

```diff
@@ -108,19 +70,40 @@ Metrics/LineLength:
 
 # Configuration parameters: CountKeywordArgs.
 Metrics/ParameterLists:
-  Max: 17
+  Max: 10
 
 Metrics/PerceivedComplexity:
   Max: 18
 
-Style/DotPosition:
+# Sometimes it's easier to read without guards
+Style/GuardClause:
   Enabled: false
 
-Style/GuardClause:
+# We allow both " and '
+Style/StringLiterals:
```

Comment:
We want this to be enabled for consistency see https://github.com/danger/danger/issues/300